ci/django.yml

```diff
@@ -0,0 +1,29 @@
+name: Django CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Run Tests
+      env: #define SECRET_KEY in Secrets
+        SECRET_KEY: ${{ secrets.SECRET_KEY }}
```

Comment:
Presumably this is https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-SECRET_KEY. Does this need to be set for running tests? If it isn't let's not do that.

Reply:
Yes this is the SECRET_KEY, it should be present to run tests.
It is currently read form the Repository Secrets.

Reply:
I'm able to run tests without a `SECRET_KEY` being specified (although I'm just trying the basic hello world). You can also specify things like `SECRET_KEY` in `tests/test_settings.py`.

Our goal is to keep the starter workflows as simple as is possible and remove anything that isn't needed. If users need to use a key, then they can set it, but I don't think its required. If you remove this I think it will get 👍 from me 😄 🙇

Reply:
Oh, okay I actually added the config just to make sure no one should push SECRET_KEYs to github.
Anyway its a users choice.
I will remove it :+1: